infrastructure/templates/template.json

```diff
@@ -1729,6 +1736,7 @@
         "NotifyPreReleaseTemplateId": "[parameters('notifyPreReleaseTemplateId')]",
         "NotifyContributorTemplateId": "[parameters('notifyContributorTemplateId')]",
         "NotifyHigherReviewersTemplateId": "[parameters('notifyHigherReviewersTemplateId')]",
```

Comment:
I could change that - the only stumbling block is the need to change the infra deploy override parameters. But since I'm changing them anyway to add the new variable, I could also change this while I'm doing it.

Meh - I'll do it as part of this PR :smile:

Reply:
Changed the description of the faffy deploy subtask ticket to include this change as part of that work.